OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -203,7 +203,7 @@ private SpawnPoint[] GetSpawnPoints(PlayerSpawner spawner)
 
         private void InitMapMarker()
         {
-            var labelText = $"GOAL: {SpeedrunState.GoalPoint.displayName.ToUpper()}";
+            var labelText = $"GOAL: {SpeedrunState.GoalPoint?.displayName.ToUpper()}";
```

Comment:
should we log when this (and maybe others) are null? prior to the null coalescence we would get an exception here in that case.